extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -36,9 +36,11 @@ import {isArray, isObject, isEnumValue} from '../../../src/types';
 import {some} from '../../../src/utils/promise';
 import {utf8Decode} from '../../../src/utils/bytes';
 import {viewerForDoc} from '../../../src/services';
+import {timerFor} from '../../../src/services';
```

Comment:
We can combine all of these into a single import.  Something like:

import {cryptoFo, resourcesForDoc, timerFor, viewerFor} from....